### PR TITLE
Tri x1 patch 1 (fixing)

### PR DESCRIPTION
**Pull request by TriX1**
can you delete pr before this

### CODE REVIEW

**Comment by TriX1** [comment]
@labster can you push and thanks to delete #499

File: LocalSettings.php [hunk 1]
```diff
@@ -1584,6 +1606,94 @@
...
+				'autopatrol' => true,
+				'patrol' => true,
+				'block' => true,
+				'protect' => true,
+				'move' => true,
+				'movefile' => false,
+				'ipblock-exempt' => false,
+			'bureaucrat' => array(
+				'edit' => true,
+				'sysop' => true,
```
Comment (revi) [inline LocalSettings.php]: This won't work.

File: LocalSettings.php [hunk 2]
```diff
@@ -1584,6 +1606,94 @@
...
+			'sysop' => array(
+				'edit' => true,
+				'nuke' => false,
+				'blockemail' =>false,
+				'deleterevision' => true,
+				'massmessage' => true,
+				'undelete' => true,
+				'autopatrol' => true,
+				'patrol' => true,
+				'ipblock-exempt' => true,
```
Comment (revi) [inline LocalSettings.php]: You don't need to specify what sysops already have.

File: LocalSettings.php [hunk 3]
```diff
@@ -1584,6 +1606,94 @@
...
+			'Founder' => array(
+				'edit' => true,
+				'sysop' => true,
+				'bureaucrat' => true,
```
Comment (revi) [inline LocalSettings.php]: again, this won't work.

File: LocalSettings.php [hunk 4]
```diff
@@ -1888,6 +1998,26 @@
...
+		 '+idtestwiki' => array(
+		 	'Founder' => array(
+		 		'autopattolled',
+		 		'bot',
+		 		'rollbacker',
+		 		'bureaucrat',
+		 		'admin',
+		 		'trusted',
+		 		'confirmed',
+		 	'admin' => array(
```
Comment (revi) [inline LocalSettings.php]: admin?sysop? choose one.

**Comment by revi** [comment]
I commented on the commit not the whole diff, but the comment still stands, so review them and fix it.

**Comment by TriX1** [comment]
Okay I can to follow you and you can change

**Comment by TriX1** [comment]
Can you merge, I cannot to commen on the diff. Why you can't merge.:)

**Comment by revi** [comment]
Because there are problems which are not trivial so I can fix it myself.

If you can't fix it, you're fine to open a new pr, but it cannot be merged as is.

**Comment by TriX1** [comment]
maybe Be late for fix.

**Comment by TriX1** [comment]
@revi @labster  @NDKilla  can you checked Thanks

**Comment by TriX1** [comment]
Hi @revi I'm close this request because I not to bothoord you. I get my passw back in accout ayst and I can to new pr. :)

**Pull request closed without merging**